documentation/book/appendix_crds.adoc

```diff
@@ -1058,13 +1058,13 @@ Used in: xref:type-KafkaBridgeStatus-{context}[`KafkaBridgeStatus`], xref:type-K
 |Property                   |Description
 |type                1.2+<.<|The unique identifier of a condition, used to distinguish between other conditions in the resource.
 |string
-|status              1.2+<.<|The status of the condition, one of True, False, Unknown.
+|status              1.2+<.<|The status of the condition, either True, False or Unknown.
```

Comment:
This is generated file. You have to do these changes in the `strimzi-kafka-operator/api/src/main/java/io/strimzi/api/kafka/model/status/Condition.java` and regenerate doc.

Reply:
@laidan6000 Do you want to try to fix it your self? Or should I open a separate PR to fix this for you?

Reply:
Please can you fix in a separate PR, @scholzj . I'll attempt to back out the change.

Reply:
@laidan6000 https://github.com/strimzi/strimzi-kafka-operator/pull/1836 - please review

Reply:
Looks good. To revert the changes to appendix_crds.adoc, can I just delete the file in GitHub? Click **...** > **Delete File**.

Reply:
I'm not sure. I would expect that to delete the whole file and not just the changes. I'm not a Git expert, but I think you should be able to:
* Fetch the data from upstream using `git fetch`
* Checkout the single file from the master branch: `git checkout origin/master documentation/book/appendix_crds.adoc`

(perhaps make a backup of the local repository before trying this - make a copy before doing this :-o)

Reply:
I'll try this... :cold_sweat: :smile:

Reply:
Give it a try. Or if you want me or whoever will be merging this can fix it for you while merging it.

Reply:
Please can you fix it during the merge?

Reply:
Sure, i will.